readme.md

```diff
@@ -124,7 +124,7 @@ By default, builds also do not use any compiler optimizations.
 Please see the release keyword argument for what compiler optimizations it will enable.
 
 ## Running the Source Code
-To start NVDA from source code, run `nvda.pyw` located in the source directory.
+To start NVDA from source code, run `nvda.pyw` located in the source directory. To view help on the arguments that nvda will accept use the `-h` or `--help` option. These arguments are also documented in the user guide. Since nvda is a Windows application (rather than command line) it is best to run it with `pythonw.exe`. However, if during development you encounter an error early in the startup of nvda you can use `python.exe` which is likely to give more information about the error.

```

Comment:
1. Please split each sentence onto its own line.
2. Comma after "will accept", "command line)" and "startup of nvda".
3. "startup of nvda": NVDA in all caps.